### PR TITLE
Fix doc of operator README for install crds

### PR DESCRIPTION
**Pull request by liangyuanpeng**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Today, It will be failed when follow the doc to create karmada instance.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: operator/README.md [hunk 1]
```diff
@@ -56,6 +56,12 @@ NAME                               READY   STATUS    RESTARTS   AGE
 karmada-operator-5b7f485c5-g5lj5   1/1     Running   0          26s
 ```
 
+### Install karmada crds
```
Comment (calvin0327) [inline operator/README.md:59]:
```suggestion
### Install Karmada operator crds
```

Comment (calvin0327) [reply operator/README.md:59]: @liangyuanpeng thanks. It should be `Karmada operator`.

Comment (liangyuanpeng) [reply operator/README.md]: Updated.

**Comment by calvin0327** [comment]
thanks
/lgtm

**Comment by calvin0327** [comment]
@liangyuanpeng Please retry the ci worlflow, you can push the branch again.

**Comment by liangyuanpeng** [comment]
>  Please retry the ci worlflow, you can push the branch again.

Thanks for your remind and I think it's not necessary to rerun ci,  the ci fail is not about this PR.

**Comment by XiShanYongYe-Chang** [comment]
The CI fail for E2E delete kind cluster failed, ask @RainbowMango to help rerun the test.
/cc @RainbowMango

**Comment by calvin0327** [comment]
@liangyuanpeng Yeah, the failed ci workflow is not relate with the PR, But it is better to follow that the ci workflow of every PR is passed.

**Comment by RainbowMango** [comment]
> The CI fail for E2E delete kind cluster failed, ask @RainbowMango to help rerun the test. /cc @RainbowMango

Done

**Pull request merged**